supfile.go

```diff
@@ -26,6 +26,7 @@ type Network struct {
 	Env       EnvList  `yaml:"env"`
 	Inventory string   `yaml:"inventory"`
 	Hosts     []string `yaml:"hosts"`
+	SSHKey    string   `yaml:"ssh-key"`
```

Comment:
`IdentityFile` is more verbose, but would match 1-1 with `~/.ssh/config` syntax, thoughts?